pkg/acme/webhook/apiserver/apiserver.go

```diff
@@ -101,14 +95,17 @@ func (c *Config) Complete() CompletedConfig {
 	completedCfg := completedConfig{
 		c.GenericConfig.Complete(),
 		&c.ExtraConfig,
-		c.restConfig,
+		c.GenericConfig.ClientConfig,
```

Comment:
instead of re-parsing the kubeconfig flag/ environment variable, we should just use the ClientConfig that was created based on the apiserver flags.

Reply:
Ok. I can see that later in the PR you have deleted the call to `InClusterConfig`.